src/distribution/laplace.rs

```diff
@@ -0,0 +1,363 @@
+use distribution::{Continuous, Univariate};
+use rand::distributions::Distribution;
+use rand::distributions::OpenClosed01;
+use rand::Rng;
+use statistics::*;
+use std::f64;
+use {Result, StatsError};
+
+/// Continuous Univariate Laplace distribution.
+/// The Laplace distribution is a distribution over the real numbers parameterized by a mean and
+/// scale parameter. The PDF is:
+///     p(x) = \frac{1}{2 * scale} \exp{- |x - mean| / scale}.
+/// <a href="http://en.wikipedia.org/wiki/Laplace_distribution">Wikipedia - Laplace distribution</a>.
```

Comment:
You can use markdown to generate your links which I think is preferable from a consistency standpoint. See [this example documentation from the binomial distribution](https://github.com/boxtown/statrs/blob/master/src/distribution/binomial.rs#L10).

It would also be nice to include an `# Examples` section (see documentation referenced above).